README.md

```diff
@@ -64,6 +64,7 @@ plugins: [
         resolve: 'gatsby-plugin-mailchimp',
         options: {
             endpoint: '', // add your MC list endpoint here; see instructions below
+            timeout: 3500, // add a timeout for the mailchimp request (3500 by default)
```

Comment:
can you change the comment to:

`// number; the amount of time, in milliseconds, that you want to allow mailchimp to respond to your request. defaults to 3500`